src/extras/primitives/registerPrimitive.js

```diff
@@ -75,6 +69,33 @@ module.exports = function registerPrimitive (name, definition) {
         }
       },
 
+      applyDefaultComponents: {
+        value: function () {
+          var self = this;
+          var defaultData = utils.extendDeep(this.defaultAttributes);
```

Comment:
yeah. if you modify this object, it will affect all other entities created from this prototype.